sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Queues/QueueListener.cs

```diff
@@ -281,6 +292,31 @@ public void Notify()
             }
         }
 
+        internal int GetMessageReceiveCount()
+        {
+            int numMessagesToReceive = _queueProcessor.QueuesOptions.BatchSize;
+
+            if (_concurrencyManager != null && _concurrencyManager.Enabled)
```

Comment:
How user knows that concurrency manager is used? It looks like BatchSize setting will be ignored in that case. Should we emit a log somewhere in init phase that this is going to happen?

Reply:
On host startup, we log ConcurrencyOptions at the host level which will show DC is enabled globally. Additionally the user will see periodic concurrency logs from DC for each of their functions using DC. So I think we're covered.